### PR TITLE
Add weekly update 2016-03-07

### PR DESCRIPTION
**Pull request by JungMinu**
Please review this post. :heart:,@nodejs/evangelism

### CODE REVIEW

**Comment by ghost** [comment]
[Travis build passed](https://travis-ci.org/nodejs/nodejs.org/builds/114240344) :+1:

**Comment by ghost** [comment]
LGTM!

File: locale/en/blog/weekly-updates/weekly-update.2016-03-07.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+We have four releases: [Node v0.10.43 (Maintenance)](https://nodejs.org/en/blog/release/v0.10.43/), [Node v0.12.11 (LTS)](https://nodejs.org/en/blog/release/v0.12.11/), [Node v5.7.1 (Stable)](https://nodejs.org/en/blog/release/v5.7.1/) and [Node v4.3.2 (LTS)](https://nodejs.org/en/blog/release/v4.3.2/). Complete changelog from previous releases can be found [on GitHub](https://github.com/nodejs/node/blob/master/CHANGELOG.md).
+
+### Initial benchmark data
+
+Initial benchmark data is now available here: https://benchmarking.nodejs.org/. 
+
+For more information on the process of how these are generated check out: https://github.com/nodejs/benchmarking/blob/master/benchmarks/README.md
+
+Get them here: https://nodejs.org/dist/v5.7.0/
```
Comment (mhdawson) [inline locale/en/blog/weekly-updates/weekly-update.2016-03-07.md]: I think this last line should not be there -> Get them here: https://nodejs.org/dist/v5.7.0/

**Comment by JungMinu** [comment]
@mhdawson Thanks, fixed :)

File: locale/en/blog/weekly-updates/weekly-update.2016-03-07.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+slug: weekly-update-2016-03-07
+layout: blog-post.hbs
+---
+
+### Node.js News
+Node v0.10.43 (Maintenance), Node v0.12.11 (LTS), Node v5.7.1 (Stable) and Node v4.3.2 (LTS) are released.
+
+### Node v0.10.43 (Maintenance), Node v0.12.11 (LTS), Node v5.7.1 (Stable) and Node v4.3.2 (LTS) Releases
+
+We have four releases: [Node v0.10.43 (Maintenance)](https://nodejs.org/en/blog/release/v0.10.43/), [Node v0.12.11 (LTS)](https://nodejs.org/en/blog/release/v0.12.11/), [Node v5.7.1 (Stable)](https://nodejs.org/en/blog/release/v5.7.1/) and [Node v4.3.2 (LTS)](https://nodejs.org/en/blog/release/v4.3.2/). Complete changelog from previous releases can be found [on GitHub](https://github.com/nodejs/node/blob/master/CHANGELOG.md).
```
Comment (thefourtheye) [inline locale/en/blog/weekly-updates/weekly-update.2016-03-07.md:16]: Can these be bullet points?

File: locale/en/blog/weekly-updates/weekly-update.2016-03-07.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+### Initial benchmark data
+
+Initial benchmark data is now available here: https://benchmarking.nodejs.org/. 
+
+For more information on the process of how these are generated check out: https://github.com/nodejs/benchmarking/blob/master/benchmarks/README.md
+
+### New Core Technical Committee (CTC) members
+
+The Core Technical Committee (CTC) added four new members to help guide Node.js core development: Evan Lucas, Rich Trott, Ali Ijaz Sheikh and Сковорода Никита Андреевич (Nikita Skovoroda).
```
Comment (thefourtheye) [inline locale/en/blog/weekly-updates/weekly-update.2016-03-07.md:26]: I believe their GitHub handles with links here would be useful

**Pull request merged**